feat: IC external integrations (#15)
SEKOIA-IO/documentation: docs/intelligence_center/integrations.md

# Intelligence Center external integrations

> The intelligence center data can be consummed using several third party integrations.

Reviewer: ```suggestion
Intelligence Center data can be consumed using several third party integrations.
```

## MISP Feed

The default feed is available as a MISP feed.

It can be added to an existing MISP instance by following [MISP's documentation](https://www.circl.lu/doc/misp/managing-feeds/).

The following field values are required for the feed to work properly:

-   Input Source: Network
-   URL: `https://api.sekoia.io/api/v2/inthreat/misp`
-   Source Format: MISP Feed

> -   Headers: `Authorization: Bearer <APIKEY>`

Reviewer: Shouldn’t we also explain how to manage APIKeys? BTW, it seems that we don’t have such a page currently.

Author: Yes I agree that it would be useful. We should create a page and add a link to it in this section.

Reviewer 2: I planned to add pages concerning all the community/user management part soon, but if someone wants to do it, fell free !

-   Enabled: `True`

You then need to make sure you have a scheduled task in place to regularly fetch the feed's content.

## OpenCTI connector

An [OpenCTI](https://www.opencti.io) connector is available to consumme a feed.

> All the instruction to run it are available at the connector github repository: https://github.com/SEKOIA-IO/opencti-connector.

Reviewer: ```suggestion
All the instruction to run it are available at the connector GitHub repository: https://github.com/SEKOIA-IO/opencti-connector.
```

## Cortex Analyser

> SEKOIA is also providing a [cortex analyzer](https://github.com/TheHive-Project/Cortex-Analyzers/tree/master/analyzers/SEKOIAIntelligenceCenter) to enrich data in [THEHIVE](https://thehive-project.org/) ecosystem.

Reviewer: ```suggestion
SEKOIA is also providing a [Cortex analyzer](https://github.com/TheHive-Project/Cortex-Analyzers/tree/master/analyzers/SEKOIAIntelligenceCenter) to enrich data in [TheHive](https://thehive-project.org/) ecosystem.
```

> To setup the analyzer please [follow the following guide](https://github.com/TheHive-Project/CortexDocs/blob/master/analyzer_requirements.md).

Author: ```suggestion
To setup the analyzer please [follow this guide](https://github.com/TheHive-Project/CortexDocs/blob/master/analyzer_requirements.md).
```

> To setup the analyzer please follow the following guide: https://github.com/TheHive-Project/CortexDocs/blob/master/analyzer_requirements.md.

Reviewer: ```suggestion
To setup the analyzer please [follow the following guide](https://github.com/TheHive-Project/CortexDocs/blob/master/analyzer_requirements.md).
```

Reviewer 2: To setup the analyzer please [follow this guide](https://github.com/TheHive-Project/CortexDocs/blob/master/analyzer_requirements.md).
